packages/lesswrong/components/users/DialogueMatchingPage.tsx

```diff
@@ -952,6 +952,8 @@ const DialogueCheckBox: React.FC<{
     if (!currentUser) return;
 
     const response = await upsertUserDialogueCheck({ targetUserId, checked: event.target.checked, checkId });
+
+    captureEvent("newDialogueCheck", {checked: response.data.upsertUserDialogueCheck.checked}) 
```

Comment:
I would extract `checked` and reuse in the different places.

So:

const checked = event.target.checked

And then use that in both the upsert call and the capture event

Reply:
yes that's a nice point. feel free to make that commit. 

Personally I really just want to get this shipped now, it's been many days. can clean up and make nicer in future. merging.